person_capture/gui_app.py

```
@@ -14503,35 +14568,38 @@ def _add_output_color_opts(opts: list[str]) -> None:
-            if output_png and not crop_after_tonemap:
-                sf = f"{seek_filter}{src},{crop},scale=in_range=limited:out_range=full,format={out_pix_fmt}"
-            else:
-                sf = f"{seek_filter}scale=in_range=limited:out_range=full,format={out_pix_fmt},{crop}"
-            cmds.append(base + ["-vf", sf] + out_once + enc)
+            scale_in_range = "full" if src_range == "full" else "limited"
+            for _variant_name, input_base, decoded_prefix in input_variants:
+                if output_png and not crop_after_tonemap:
+                    sf = f"{decoded_prefix}{src},{crop},scale=in_range={scale_in_range}:out_range=full,format={out_pix_fmt}"
+                else:
+                    sf = f"{decoded_prefix}scale=in_range={scale_in_range}:out_range=full,format={out_pix_fmt},{crop}"
+                cmds.append(input_base + ["-vf", sf] + out_once + enc)
```

Ruff (0.15.12)

[warning] Consider `[*input_base, "-vf", sf, *out_once, *enc]` instead of concatenation

Replace with `[*input_base, "-vf", sf, *out_once, *enc]`

(RUF005)